<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>291</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QPushButton" name="drinkButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Drink Potion</string>
     </property>
    </widget>
    <widget class="QComboBox" name="potionsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>46</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string>Potions:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="castButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cast Spell</string>
    </property>
   </widget>
   <widget class="QPushButton" name="damageButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Damage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeName">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="changeRace">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Race</string>
    </property>
   </widget>
   <widget class="QComboBox" name="racesList">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>textEdit</zorder>
   <zorder>castButton</zorder>
   <zorder>damageButton</zorder>
   <zorder>changeName</zorder>
   <zorder>lineEdit</zorder>
   <zorder>changeRace</zorder>
   <zorder>racesList</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
